primitives/arithmetic/src/traits.rs

```diff
@@ -26,6 +26,8 @@ pub use num_traits::{
 use sp_std::ops::{
 	Add, AddAssign, Div, DivAssign, Mul, MulAssign, Rem, RemAssign, Shl, Shr, Sub, SubAssign,
 };
+use funty::AtMost32;
```

Comment:
This is easy enough to make in-house, why add a dependency for it? 

Generally, we try to limit dependencies in the runtime and avoid adding new ones unless if really needed.

Reply:
Your point makes complete sense, @kianenigma. I think, we don't even need to have a `AtMost32` trait bound, `Into<u32> + Unsigned` should do the job.